.github/workflows/ci-tests.yml

```diff
@@ -12,11 +12,9 @@ jobs:
     uses: ./.github/workflows/linters.yml
 
   build:
-    if: ${{ contains(github.event.pull_request.labels.*.name, 'ok-to-test') }}
```

Comment:
why are you removing this?

Reply:
After this change, CI tests will only be executed after a merge in main. So this check won't be required.

Reply:
I don't understand, the trigger event is still pull_request_target

Reply:
Also, why do you want that change?, secrets can be acceded from a PR too

Reply:
Yes you are right. According to the docs [here](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request_target), `pull_request_target` seems to be providing gh-secret access to the fork PRs (for example, openshift cluster secrets working against PRs). Not having a way to [test it](https://github.com/kube-burner/kube-burner-ocp/actions/runs/11223524706/job/31198394006?pr=116) before the merge confused me. 

So technically after this PR is merged, the follow up PRs should be able to succeed on the current failing CI test.
While `pull_request_target` provides untrusted access to the forks, `pull_request` will help providing access to the PRs only against the main stream repo not for forks. Previous experiment [here](https://github.com/cloud-bulldozer/orion/actions/runs/10304354144)

Reply:
Changed and updated the PR as suggested.